data/mods/ssb/abilities.ts

```diff
@@ -67,6 +67,37 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		},
 	},
 
+	// aegii
+	newstage: {
+		desc: "Stance Change; Haze, Heal Bell and Embargo start on switch in.",
+		shortDesc: "Stance Change; Haze, Heal Bell and Embargo start on switch in.",
+		onBeforeMovePriority: 0.5,
+		onBeforeMove(attacker, defender, move) {
+			if (attacker.species.baseSpecies !== 'Aegislash' || attacker.transformed) return;
+			if (move.category === 'Status' && move.id !== 'kingsshield' && move.id !== 'kshield') return;
+			const targetForme = (move.id === 'kingsshield' || move.id === 'kshield' ? 'Aegislash' : 'Aegislash-Blade');
+			if (attacker.species.name !== targetForme) attacker.formeChange(targetForme);
+		},
+		onStart(pokemon) {
+			this.add('-clearallboost');
+			for (const pokemon of this.getAllActive()) {
+				pokemon.clearBoosts();
+			}
+			this.add('-activate', pokemon, 'move: Heal Bell');
+			const side = pokemon.side;
+			let success = false;
+			for (const ally of side.pokemon) {
+				if (ally !== pokemon && ally.hasAbility('soundproof')) continue;
+				if (ally.cureStatus()) success = true;
+			}
+			if (pokemon.side.foe.active[0]) {
+				pokemon.side.foe.active[0].addVolatile('embargo');
+				this.add('-start', pokemon, 'Embargo');
+			}
+		},
```

Comment:
honestly i'd rather if you use all 3 moves simultaneously a la Zodiax

Reply:
I feel going this way feels better cause then it feels like an ability instead of simply using the moves

Reply:
alright fair enough. That looks good